src/etc/inc/upgrade_config.inc

```diff
@@ -5493,6 +5493,17 @@ function upgrade_172_to_173() {
 	}
 }
 
+/*
+ * Delete old Dynamic DNS dnssec-keygen style files,
+ * they are replaced with a simpler single ddns-confgen style file.
+ */
+function upgrade_173_to_174() {
```

Comment:
As long as you're doing upgrade code, you may as well unset the `keytype` field on the rfc2136 entries since it was removed.